os/include/tinyara/bluetooth/bt_hci.h

```diff
@@ -219,7 +219,7 @@
 #define BT_HCI_LE_ADV_DISABLE                   0x00
 #define BT_HCI_LE_ADV_ENABLE                    0x01
 #define CONFIG_BT_ID_MAX                        2
-#define HCI_MAX_NAME_LENGTH             248
+#define HCI_MAX_NAME_LENGTH                     65
```

Comment:
Is this change required? When we use '248' length, which problem is occured?

Reply:
* BLUETOOTH_MAX_FRAMELEN
 * Maximum amount of data that can fit in a buffer.
 *
 * The biggest foreseeable buffer size requirement right now comes from
 * the Bluetooth 4.2 SMP MTU which is 65. This then become 65 + 4 (L2CAP
 * header) + 4 (ACL header) + 1 (H4 header) = 74. This also covers the
 * biggest HCI commands and events which are a bit under the 70 byte
 * mark.
 */

#define BLUETOOTH_MAX_FRAMELEN  (BLUETOOTH_MAX_MTU + BLUETOOTH_MAX_HDRLEN)

#define BLUETOOTH_MAX_MTU       70
#define BLUETOOTH_L2CAP_HDRLEN  4       /* Size of L2CAP header */
#define BLUETOOTH_ACL_HDRLEN    4       /* Size of ACL header */
#define BLUETOOTH_H4_HDRLEN     1       /* Size of H4 header */

so bt_buf could be maximum 79 byte.
when I was testing write local name with 248 byte, that time, there was crash happening.
So I found out that we can not have local name with 248 byte, we can have local name with 65 byte and together with header (9 byte), bt_buf would be 74 byte.